PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.API/Program.cs

```diff
@@ -18,8 +104,17 @@
 
 app.UseHttpsRedirection();
 
+app.UseAuthentication();
+app.UseAuthorization();
+
 app.UseAuthorization();
 
 app.MapControllers();
 
+app.UseCors(x => {
+    x.WithHeaders().AllowAnyHeader();
+    x.WithOrigins("http://localhost:3000");
```

Comment:
If you want to configure CORS, they should be grabbed from appsettings. Anyway, for this task it's enough to allow any origin